generic/tests/cfg/netperf.cfg

```diff
@@ -63,6 +63,8 @@
     ppc64le:
         setup_cmd = "cd /tmp && rm -rf netperf-2.6.0 && tar xvfj netperf-2.6.0.tar.bz2 && cd netperf-2.6.0 && ./configure --build=ppc64le --enable-burst --enable-demo=yes  && make"
     log_hostinfo_script = scripts/rh_perf_log_hostinfo_script.sh
+    host_tuned_profile = "tuned-adm profile virtual-host"
+    guest_tuned_profile = "tuned-adm profile virtual-guest"
```

Comment:
Recommend move it to `Linux:` section if it is linux only param.

Reply:
update done, please help review. Thanks,